### PR TITLE
Feature/enhance central repo description

### PR DESCRIPTION
**Pull request by frivieremicroej**
- a selection of production-grade modules
- bidirectionnal link to sdk eula
- remove MicroEJ
- "various licenses" as in the schematics

### CODE REVIEW

File: SDKUserGuide/repository.rst [hunk 1]
```diff
@@ -8,33 +8,35 @@ This chapter describes the :ref:`module repositories <module_repository>` provid
...
 
-MicroEJ Central Repository
-==========================
+Central Repository
+==================
 
-The MicroEJ Central Repository is the module repository distributed and maintained by
-MicroEJ Corp. It contains Foundation Library APIs and numerous Add-On Libraries. 
+The Central Repository is the module repository distributed and maintained by MicroEJ Corp. 
+It contains a selection of production-grade modules such as Foundation Libraries, Add-On Libraries and :ref:`Packs <pack_overview>`.
```
Comment (tdelhomenie-microej) [inline SDKUserGuide/repository.rst:15]: We could add links for Addon and Foundation Libraries to https://docs.microej.com/en/latest/ApplicationDeveloperGuide/libraries.html

File: SDKUserGuide/repository.rst [hunk 2]
```diff
@@ -8,33 +8,35 @@ This chapter describes the :ref:`module repositories <module_repository>` provid
...
 
 Use
 ---
 
-By default, MicroEJ SDK is configured to connect the MicroEJ Central Repository. 
+By default, the SDK is configured to connect the online Central Repository.
```
Comment (tdelhomenie-microej) [inline SDKUserGuide/repository.rst:20]:
```suggestion
By default, the SDK is configured to connect to the online Central Repository. 
```

File: SDKUserGuide/repository.rst [hunk 3]
```diff
@@ -53,13 +55,13 @@ MicroEJ Corp. contributes to the developer repository in the following cases:
...
 - Incubating Libraries,
-- Former MicroEJ Central Repository versions,
+- Former Central Repository versions,
 - Hardware specific modules.
 
 Use
 ---
 
-By default, MicroEJ SDK is configured to connect the developer repository [#warning_require_sdk_5_4]_.
+By default, the SDK is configured to connect the developer repository [#warning_require_sdk_5_4]_.
```
Comment (tdelhomenie-microej) [inline SDKUserGuide/repository.rst:64]:
```suggestion
By default, the SDK is configured to connect to the developer repository [#warning_require_sdk_5_4]_.
```

File: SDKUserGuide/repository.rst (hunk 1 above)
Comment (frivieremicroej) [reply SDKUserGuide/repository.rst:15]: done

**Review by tdelhomenie-microej — APPROVED** [review]

**Pull request merged**